generators/common/templates/.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+npm exec --no lint-staged
```

Comment:
The `npm exec` command introduced at npm v7
By JHipster [installation docs](https://www.jhipster.tech/installation/): 
> please use an LTS 64-bit version, non-LTS versions are not supported

The latest node LTS version is: 14.17.6 (includes **npm 6.14.15**)
npm 6.14.15 does not support `exec`

Am I missing something?

Reply:
I think you're right. It can explain some errors I encounter recently with Husky. Can you open a specific ticket plz, so we can track this and try to find a fix?

Reply:
https://github.com/jhipster/generator-jhipster/issues/16336